embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/EmbraceFeatureRegistry.kt

```diff
@@ -0,0 +1,15 @@
+package io.embrace.android.embracesdk.arch
+
+import io.embrace.android.embracesdk.arch.datasource.DataSourceState
+
+/**
+ * Registry for all features whose instrumentation should be orchestrated by the Embrace SDK.
+ */
+internal interface EmbraceFeatureRegistry {
```

Comment:
I anticipate exposing this as part of the internal API between modules that allows us to add automatic instrumentation to the Embrace SDK